help.go

```diff
@@ -12,12 +14,65 @@ import (
 	"github.com/go-task/task/v3/internal/logger"
 )
 
+// ListOptions collects list-related options
+type ListOptions struct {
+	ListOnlyTasksWithDescriptions bool
+	ListAllTasks                  bool
+	FormatTaskListAsJSON          bool
+}
+
+// NewListOptions creates a new ListOptions instance
+func NewListOptions(list, listAll, listAsJson bool) ListOptions {
+	return ListOptions{
+		ListOnlyTasksWithDescriptions: list,
+		ListAllTasks:                  listAll,
+		FormatTaskListAsJSON:          listAsJson,
+	}
+}
+
+// ShouldListTasks returns true if one of the options to list tasks has been set to true
+func (o ListOptions) ShouldListTasks() bool {
+	return o.ListOnlyTasksWithDescriptions || o.ListAllTasks
+}
+
+// Validate validates that the collection of list-related options are in a valid configuration
+func (o ListOptions) Validate() interface{} {
+	if o.ListOnlyTasksWithDescriptions && o.ListAllTasks {
+		return fmt.Errorf("cannot use --list and --list-all at the same time")
```

Comment:
We prepend `task: ` to errors that are from Task itself.

This a common Go pattern to know where an error came from. And it's useful here because Tasks prints outputs from other CLIs as well.

```suggestion
		return fmt.Errorf("task: cannot use --list and --list-all at the same time")
```

Reply:
fixed in a rebased commit